android/capacitor/src/main/java/com/getcapacitor/Plugin.java

```diff
@@ -168,14 +167,32 @@ public PluginCall getSavedCall() {
         return this.savedLastCall;
     }
 
+    /**
+     * Get the config options for this plugin.
+     *
+     * @return a config object representing the plugin config options
+     */
+    public PluginConfig getConfig() {
+        return bridge.getConfig().getPluginConfiguration(handle.getId());
```

Comment:
Would it make sense to return an empty `PluginConfig` to avoid possible null exceptions?

Reply:
Thanks